contents/testing/writing-integration-test.md

```diff
@@ -0,0 +1,143 @@
+<frontmatter>
+  title: Integration Testing
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Integration Testing
+
+**Author(s): [Jacob Li PengCheng](https://github.com/jacoblipech)**
+
+Reviewers:
+
+## What is integration testing?
+
+Integration testing is a part of software testing where individual units of your application are combined and tested as a group. This usually happens after [unit testing stage](http://softwaretestingfundamentals.com/unit-testing/) and before the [system testing stage](https://www.guru99.com/system-testing.html).
+
+## Why is integration testing important to SE?
+
+In a software application, each software module is usually designed and unit tested by different programmers. Hence, with different understanding of the software requirements, integration testing is necessary to ensure that software modules work in unity and expose any faults in the interaction between different parts of the units. 
```

Comment:
different parts of the units -> different units?

parts of the units sound like the different parts in each individual unit to me 😅